pkg/predict/input.go

```diff
@@ -1,20 +1,23 @@
 package predict
 
 import (
+	"encoding/json"
+	"fmt"
+	"mime"
 	"os"
 	"path/filepath"
 	"strings"
 
-	"github.com/mitchellh/go-homedir"
 	"github.com/vincent-petithory/dataurl"
 
+	"github.com/mitchellh/go-homedir"
 	"github.com/replicate/cog/pkg/util/console"
-	"github.com/replicate/cog/pkg/util/mime"
 )
 
 type Input struct {
 	String *string
 	File   *string
+	Array  *[]interface{}
```

Comment:
Two small points: 

1. it's nice to use `any` rather than `interface{}` these days
2. this doesn't need to be a pointer -- slices are already pointer types